detection_rules/beats.py

```diff
@@ -174,6 +174,9 @@ def get_beats_sub_schema(schema: dict, beat: str, module: str, *datasets: str):
         dataset_dir = module_dir.get("folders", {}).get(dataset, {})
         flattened.extend(get_field_schema(dataset_dir, prefix=module + ".", include_common=True))
 
+    # we also need to capture (beta?) fields which are directly within the module _meta.files.fields
+    flattened.extend(get_field_schema(module_dir, include_common=True))
```

Comment:
hmm... is this the right call?
IIRC, this blew up the schema significantly before and added more fields that shouldn't be there.
do you have a sense what new fields and how many were added?

Reply:
not for every module, but `threatintel.indicator*` (and some other `threatintel.*` specifically was _entirely_ embedded under this. Not sure if this is actually where beta fields are stuffed, but it seems consistent. Without it, validation will fail on threatintel.indicator*

Reply:
yeah, i'm following that this is the only way for this rule to be validated.
we should double check the impact on the schema and validation in general. i think for other beats and modules, this brings in more fields than is correct

Reply:
Here is a full dump of what's included per beats/module:

[beats-module-base-fields.txt](https://github.com/elastic/detection-rules/files/6361331/beats-module-base-fields.txt)

Reply:
👍